quickstarts/synthetics/free_tier/config.yml

```diff
@@ -0,0 +1,32 @@
+id: 260027d0-9d8f-4a35-a676-73b5ee6c0d2d
+# Name of the quickstart (required)
+slug: free-tier
+
+# Displayed in the UI (required)
+title: Free Teir
+
+# Long-form description of the quickstart (required)
+description: |
+  # We find Free tier
```

Comment:
This could use a little more descriptive. What does this quickstart provide? How will this benefit other people.

Reply:
Same with the longer form `summary` field.